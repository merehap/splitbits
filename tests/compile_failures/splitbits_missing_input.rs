use splitbits::splitbits;

fn main() {
    splitbits!(min=u8, "aaaabbbb");
}
