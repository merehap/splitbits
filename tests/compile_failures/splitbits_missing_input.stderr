error: proc macro panicked
 --> tests/compile_failures/splitbits_missing_input.rs:4:5
  |
4 |     splitbits!(min=u8, "aaaabbbb");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Either an input or template was missing, but found a setting instead.
